examples/encoder_example.c

```diff
@@ -750,16 +769,21 @@ int main(int argc, char **argv) {
     if (video_time == -1) continue;
     video_kbps = (int)rint(video_bytesout*8*0.001/video_time);
     time_base = video_time;
+    current_frame_no = time_base*video_fps;
     if (interactive) {
       fprintf(stderr, "\r");
     }
     else {
       fprintf(stderr, "\n");
     }
+    t1 = clock();
+    time_spent = (double)(t1 - t0) / CLOCKS_PER_SEC;
     fprintf(stderr,
-     "     %i:%02i:%02i.%02i video: %ikbps          ",
+     "     %i:%02i:%02i.%02i video: %ikbps - Frame %i - %f FPS - %f FPM     ",
      (int)time_base/3600, ((int)time_base/60)%60, (int)time_base % 60,
-     (int)(time_base*100 - (long)time_base*100), video_kbps);
+     (int)(time_base*100 - (long)time_base*100), video_kbps, current_frame_no, 
+     (current_frame_no)/time_spent, 
+     (current_frame_no)/time_spent * 60);
```

Comment:
Operators with precedence greater than + / - have no spaces. So no spaces around *.

(yes, some of the code around it is old and doesn't follow the style guide)